packages/client/modules/demo/initDB.ts

```diff
@@ -420,6 +420,49 @@ const initNewMeeting = (organization, teamMembers, meetingMembers) => {
   } as Partial<IRetrospectiveMeeting>
 }
 
+const initViewer = (user) => {
+  return {
+    viewer: {
+      ...user,
+      userOnTeam: {
+        ...user
+      },
+      assigneeTeamMember: {
```

Comment:
-1 `assigneeTeamMember` gets 2 arguments: `userId` and `teamId`. `teamId` we can ignore, but the `integrations` and `preferredName` are dependent on the task and thus different `userId`s are passed here
https://github.com/ParabolInc/parabol/blob/cdef5e550ebbd5e076744ffeab6307df7a2b476a/packages/client/components/TaskFooterIntegrateMenu.tsx#L59
It should be ok to just set the `integrations` to null if `userId !== user[0].id`.